src/models/mlp.py

```diff
@@ -0,0 +1,98 @@
+from typing import List
+from torch.nn import Module, Linear, ReLU, Dropout, Sigmoid, Sequential, BCELoss
+import torch.optim as optim
+from skorch import NeuralNetClassifier
+
+class MLPModule(Module):
+    """
+    Multi-layer Perceptron classifier for longitudinal data with skorch 
+
+    Args:
+        input_size (int): Number of input features
+        hidden_sizes (List[int]): List of sizes for hidden layers
+        output_size (int): Number of output units
+        dropout_rate (float): Dropout rate for regularization
+    """
+    def __init__(self, input_size: int, hidden_sizes: List[int], output_size: int, dropout_rate: float):
+        super(MLPModule, self).__init__()
+        
+        layers: List[Module] = []
+        in_size: int = input_size
+        
+        for h_size in hidden_sizes:
+            layers.append(Linear(in_size, h_size))
+            layers.append(ReLU())
+            layers.append(Dropout(dropout_rate))
+            in_size = h_size
+
+        layers.append(Linear(in_size, output_size))
+        layers.append(Sigmoid())
+        self.model: Module = Sequential(*layers)
+
+    def forward(self, X):
+        """
+        Forward pass through the network.
+
+        Args:
+            X (torch.Tensor): Input tensor.
+
+        Returns:
+            torch.Tensor: Output tensor.
+        """
+        return self.model(X)
+
+class MLP:
+    """
+    Wrapper class for skorch NeuralNetClassifier with MLPModule.
```

Comment:
Just curious. Cannot you inherit from NeuralNetClassifier in `class MLP` ? So that `class MLP(NeuralNetClassifier)` is actually really a wrapper. To try it out.